bitcoin/secp256k1_main.py

```diff
@@ -311,16 +302,18 @@ def ecdsa_raw_sign(msg,
         ndata = ffi.new("char [32]", usenonce)
         usenonce = (nf, ndata)
     if formsg:
-        sig = newpriv.ecdsa_sign_recoverable(msg, raw=rawmsg)
-        s, rid = newpriv.ecdsa_recoverable_serialize(sig)
+        if rawmsg:
+            sig = newpriv.sign_recoverable(msg, hasher=None)
```

Comment:
I guess we don't really need this clause, because `formsg` is supposed to be a flag for message signing, and that will always be hashed.